azurecompute/pom.xml

```diff
@@ -56,6 +56,11 @@
       <version>${project.parent.version}</version>
     </dependency>
     <dependency>
+      <groupId>org.apache.jclouds.driver</groupId>
+      <artifactId>jclouds-sshj</artifactId>
+      <version>${project.parent.version}</version>
```

Comment:
Without `jclouds-sshj` I don't have classes like as `org.bouncycastle.openssl.PEMKeyPair`, `org.bouncycastle.openssl.PEMParser` and `org.bouncycastle.openssl.jcajce.JcaPEMKeyConverter` even if I include `jclouds-bouncycastle`.

Does this mean that I need to stop using such classes and revert to manual parsing?